webrtc.html

```diff
@@ -6907,10 +6907,11 @@ <h2>Methods</h2>
       respectively, including the most recent time a
       packet that the source contributed to was played out. The browser MUST
       keep information from RTP packets received in the previous 10 seconds.
-      When the first audio frame contained in an RTP packet is delivered to the
-      <code><a>RTCRtpReceiver</a></code>'s <code><a>MediaStreamTrack</a></code>
-      for playout, the user agent MUST queue a task to update the relevant
-      information for the <code><a>RTCRtpContributingSource</a></code> and
+      When the first audio or video frame contained in an RTP packet is
+      delivered to the <code><a>RTCRtpReceiver</a></code>'s
+      <code><a>MediaStreamTrack</a></code> for playout, the user agent MUST
```

Comment:
This would mandate browsers MUST implement this for video as well as audio. Wouldn't this require list discussion?

Firefox currently implements this API for audio only. cc @na-g

Reply:
getContributingSources() was first implemented in ORTC both for audio and video.  So not sure how this was interpretted as "audio only" in WebRTC 1.0.

Reply:
FWIW I'd agree to @jan-ivar here. I _think_ we made a conscious decision to only do audio once, and should discuss if we revisit.

Reply:
This is basically just exposing RTP timestamps, which is not inherently audio-only, and is useful for group calls. I do propose we mandate browsers must implement it, I also think it would be low hanging fruit to do so (in third_party/webrtc it's mostly a few lines of copy-paste). The "source" and "timestamp" is enough to:
- Know if stream is active or not in in a way that _muted_ inherently does not tell you.
- Timestamps in real-time in a way that getStats() cannot give you (too heavy-weight).
- For large meetings where streams are not always active (whether SSRCs or CSRCs), this allows you to know which streams should be prioritized e.g. shown in the UI.

Reply:
I agree on the usage, and I personally have no opinion either way. If implementors doesn't object it sounds like a good idea to add IMHO. But I think it's a decision for @aboba and @alvestrand whether this should just be added, or if any discussion is needed.

Reply:
> This is basically just exposing RTP timestamps

It's not the RTP timestamp, it's playout time. See https://github.com/w3c/webrtc-pc/issues/1983#issuecomment-425080297

Reply:
My bad, it's playout timestamp (not RTP timestamp). This PR already reflects this based on existing language.

Based on follow-up discussion at https://github.com/w3c/webrtc-pc/issues/1983, is there anything that needs to change with the current PR or should we merge it?

Reply:
Lgtm.